services/spar/src/Spar/Scim/User.hs

```diff
@@ -222,7 +219,9 @@ validateScimUser'
   -> m ValidScimUser
 validateScimUser' idp richInfoLimit user = do
     uref :: SAML.UserRef <- mkUserRef idp (Scim.externalId user)
-    handl <- validateHandle (Scim.userName user)
+    handl <- validateHandle . Text.toLower . Scim.userName $ user
+        -- FUTUREWORK: 'Scim.userName' should be case insensitive; then the toLower here would
```

Comment:
I'm not sure exactly what this comment means. why is `toLower` brittle?

Reply:
we need to make sure it's always called, so it would be easier if there was no need to call it at all.  that would result in code that is less rigid and easy to break.

does this explain it better?  can you suggest a comment that makes this clearer, but is not too long?